database/sql/V072__auction_solution_orders.sql

```diff
@@ -0,0 +1,36 @@
+-- Table to store all proposed solutions for an auction, received from solvers during competition time.
+-- A single auction can have multiple solutions, and each solution can contain multiple orders.
+-- Order is not allowed to be part of more than one solution within the same auction.
```

Comment:
> Order is not allowed to be part of more than one solution within the same auction.

This is only true for `winning` solutions, no? There is no reason why multiple solvers should not be able to propose solutions for the same order.
And there are even ideas floating around where could just tell all solvers to submit their solution with a tip proportional to their score to let block builders figure out how to most optimally fit these solutions into a block.

Reply:
> This is only true for winning solutions, no?

True. Refactored the tables so I don't think this is an issue anymore.